Server.js

```diff
@@ -50,7 +20,40 @@ function IsJSON(data)
   return true
 }
 
+function GetFullTime(){
+  var date_obj = new Date()
+  var day = date_obj.getDate()
+  var month = date_obj.getMonth() + 1
+  var year = date_obj.getFullYear()
+  var time = date_obj.getHours() + ":" + date_obj.getMinutes() + ":" + date_obj.getSeconds()
+
+  return day + "-" + month + "-" + year + ", " + time
+}
+
+app.get('/api/tasks/:id', function(req, res){
+  var data = fs.readFileSync("TasksFile.JSON")
+  var JSON_obj_file = JSON.parse(data)
+
+  var task_id = req.params.id
+
+  res.send(JSON_obj_file[task_id])
+})
+
+app.get('/api/tasks', function(req, res){
+  var data = fs.readFileSync("TasksFile.JSON")
+  var JSON_obj_file = JSON.parse(data)
+
+  var tasks_Array = []
+
+  for (const task in JSON_obj_file) {
```

Comment:
consider both:
a. use map
b. remove this loop and return just the dict itself for improving performance